beacon-chain/rpc/eth/shared/structs.go

```diff
@@ -24,6 +24,10 @@ type AttestationData struct {
 	Target          *Checkpoint `json:"target" validate:"required"`
 }
 
+type BlockRootContainer struct {
```

Comment:
I don't think we need this type. I doubt it will ever get reused. I would rewrite `BlockRootResponse` as
```
type BlockRootResponse struct {
	Data *struct {
		Root string `json:"root"`
	} `json:"data"`
	ExecutionOptimistic bool `json:"execution_optimistic"`
	Finalized           bool `json:"finalized"`
}
```